src/__tests__/fieldValidators.js

```diff
@@ -156,6 +157,28 @@ it('testEmailFormat', () => {
     expect(actual).toBe(expected);
 });
 
+it('testLinkedinUrlFormat', () => {
+    let actual = testLinkedinUrlFormat('https://www.linkedin.com/in/url-test/');
+    let expected = null;
+    expect(actual).toBe(expected);
+
+    actual = testLinkedinUrlFormat('Bob');
+    expected = 'notLinkedinUrl';
+    expect(actual).toBe(expected);
+
+    actual = testLinkedinUrlFormat(null);
+    expected = null;
```

Comment:
Why would you expect null if there's no value ?

Reply:
It doesn't check if the field is required, it's the job of `validateRequired`

Reply:
Sounds good to me ;)